app/controllers/idv/hybrid_mobile/document_capture_controller.rb

```diff
@@ -43,13 +44,26 @@ def update
         end
       end
 
+      # Given that the start of the IPP flow is in the TrueID doc_auth React app,
+      # we need a generic, direct way to start the IPP flow
+      def direct_in_person
+        attributes = {
+          remaining_submit_attempts: rate_limiter.remaining_count,
+          flow_path: :hybrid,
+        }.merge(ab_test_analytics_buckets)
+        analytics.idv_in_person_direct_start(**attributes)
+
+        redirect_to idv_hybrid_mobile_document_capture_url(step: :idv_doc_auth)
+      end
+
       def extra_view_variables
         {
           flow_path: 'hybrid',
           mock_client: doc_auth_vendor == 'mock',
           document_capture_session_uuid: document_capture_session_uuid,
           failure_to_proof_url: return_to_sp_failure_to_proof_url(step: 'document_capture'),
           doc_auth_selfie_capture: resolved_authn_context_result.facial_match?,
+          skip_doc_auth_from_socure_hybrid: @skip_doc_auth_from_socure_hybrid,
```

Comment:
```suggestion
          skip_doc_auth_from_socure: @skip_doc_auth_from_socure,
```

this is a great attribute addition ... can we add this same attribute to the standard flow? also, should we remove the flow from the name since the flow is a present field?

Reply:
I added the `skip_doc_auth_from_socure to standard`, but can't avoid the `skip_doc_auth_from_handoff` since that is set as a side effect of `Idv::DocumentCaptureController#allow_direct_ipp?`

Reply:
thanks @solipet 🙏🏿 

it seems like we could conditionally set skip_doc_auth_from_hybrid_handoff, i have create an [issue](https://cm-jira.usa.gov/browse/LG-15491) to carefully address this later 👍🏿